.travis.yml

```diff
@@ -6,7 +6,6 @@ os:
     - linux
 cache:
     directories:
-        - compiler/.cabal-sandbox
```

Comment:
Does `.stack-work` replace `.cabal-sandbox`? Might be a good idea to cache that.